build.sh

```diff
@@ -225,6 +225,10 @@ while [[ $# -gt 0 ]]; do
         NO_JIT="-DNO_JIT_SH=1"
         ;;
 
+    --intl-icu)
```

Comment:
Yes, that makes sense. Since on Linux the presence of Intl will imply an ICU implementation, `--intl-icu` is redundant.